### DISCUSSION TITLE
Sending results to MS Teams

### DISCUSSION THREAD

**Discussion by SaulGoode77** [LironEr/cypress-mochawesome-reporter#161] [General]
This reported is definitely the best cypress reporter out there.  I just started using it with cypress-parallel and I can't believe how well it works.  Was so easy to setup and worked on the first try (Thanks to example doc).

The only thing I'm missing now, is being able to send the test results to MS Teams from Jenkins/github actions.  Is there a simple way of doing that with this reporter, or would I have to use multiple-reporters and find some library for a different reporter?

Thanks

**Comment by LironEr** [comment]
Glad you find this reporter useful (:
From what I remember you can use multiple reporters with cypress-parallel, so that's an option, or you can get the final JSON file this reporter creates (after merging all JSON files) and send it somewhere.

Let me know if you need help setting this up, I can try create an example if neeeded.
